OracleTuxedo/art_runtime/dockerfiles/12.2.2/Dockerfile.template

```diff
@@ -0,0 +1,84 @@
+#
+# Dockerfile template for Tuxedo ART 12.2.2
+# 
+# For Tuxedo 12.2.2, you need JRE installed first, so before you can build this image, you must download the Oracle Server JRE binary and drop in folder OracleJDK/java-8 and build that image.
+#   cd OracleJDK/java-8
+#   sh build.sh
+
+# Pull base image
+FROM oracle/tuxedo:12.2.2
+
+MAINTAINER Judy Liu <judy.liu@oracle.com>
+
+# Create the installation directory tree and user oracle with a password of oracle
+USER root
+RUN yum -y install perl perl-CPAN ksh mksh sudo make vim tar net-tools dos2unix tk tcl-devel tk-devel gtk2 expect libXtst gnome-desktop xorg-x11-xauth dejavu-lgc-sans-fonts liberation-sans-fonts libcanberra-gtk2 PackageKit-gtk3-module libaio openssh-server openssh-client rsync; yum -y clean all 
+
+#Set environments
+ENV ORACLE_HOME=/usr/lib/oracle/12.2/client64 \
+    JAVA_HOME=/usr/java/default \
+    PATH=/usr/java/default/bin:$PATH \
```

Comment:
The Tuxedo Core image from which this image is derived is based on the Server JRE image, which should already have the java binaries in the `PATH` via the `alternatives` system. You shouldn't need to change the `PATH` environment variable here.